addons/binding/org.openhab.binding.network/META-INF/MANIFEST.MF

```diff
@@ -27,5 +27,6 @@ Import-Package:
  org.openhab.binding.network,
  org.openhab.binding.network.handler,
  org.osgi.service.component,
+ org.osgi.service.component.annotations;resolution:=optional,

```

Comment:
Should this also be reverted https://github.com/openhab/openhab-core/pull/237

Reply:
yes.